selfdrive/loggerd/loggerd.cc

```diff
@@ -271,11 +271,16 @@ void encoder_thread(int cam_idx) {
         int out_id = encoders[0]->encode_frame(buf->y, buf->u, buf->v,
                                                buf->width, buf->height,
                                                &out_segment, extra.timestamp_eof);
+        if (out_id == -1) {
+          continue;
+        }
         if (encoders.size() > 1) {
           int out_segment_alt = -1;
-          encoders[1]->encode_frame(buf->y, buf->u, buf->v,
-                                    buf->width, buf->height,
-                                    &out_segment_alt, extra.timestamp_eof);
+          if (-1 == encoders[1]->encode_frame(buf->y, buf->u, buf->v,
```

Comment:
Maybe we should ignore a failure of the alternate encoders? Now we can successfully encode the normal HEVC, but not send out a corresponding encodeIdx message.

Reply:
Maybe a for loop is better. can solve these two problems

Reply:
If you think this is not good, I will revert back